extra/generate-display-names/Makefile

```diff
@@ -1,8 +1,11 @@
 CC := gcc
```

Comment:
TBH I don't quite understand when and how this is invoked. It doesn't seem to be in the build system. Could we not just disable this for the other architectures?

Reply:
Thanks @egli , I will do it.